### PR TITLE
Update FAQ: warn_for_others

### PR DESCRIPTION
**Pull request by larswmh**
This PR solves #1939. The images at the end of the article have been removed and a link to both iOS and Android screenshots has been added.

EN:
image
DE:
image

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@larswmh 
Thank you for picking this up!

1) You could also delete the images warn_for_others*.png from https://github.com/corona-warn-app/cwa-website/tree/master/src/assets/img/faq

2) Would you like to make the change to refer to the Status screen like you did for other FAQs? I'll mark this up inline as well.

File: src/data/faq.json [hunk 1]
```diff
@@ -1436,8 +1436,7 @@
...
                             "To alert other guests using the CWA as an event organizer, please follow these steps:",
                             "<ul><li>If you subsequently become aware of the presence of a positively tested person at your event, and the health department has not yet contacted you as an event organizer, contact the health department. The health department can issue you a TAN if necessary.</li><li>Click 'Create QR Code' on the CWA home page.</li><li>On the 'My QR Codes' page, click the three-dot menu in the upper right corner and then click the 'Warn for Others' menu item.</li><li>Click 'Continue' on the info screen.</li><li>Select an event for which you would like to warn others for an infected person. Or restore a deleted event by scanning the event's QR code again. Click 'Continue'.</li><li>Indicate the length of stay of the person who tested positive for whom you want to warn. You can get this information from the health department. Click on 'Continue'.</li><li>Enter the 10-digit TAN that you were given. Click on 'Send warning'.</li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq.json:1438]: Replace "on the CWA home page" with "on the Status tab ('Start Screen' before version 2.12) of your CWA"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Additional comments in-line.

Thanks again!

File: src/data/faq_de.json [hunk 2]
```diff
@@ -1445,8 +1445,7 @@
...
                             "<ul><li>Erhalten Sie im Nachhinein Kenntnis von der Gegenwart einer positiv getesteten Person auf Ihrer Veranstaltung, und hat das Gesundheitsamt Sie als Organisator einer Veranstaltung noch nicht kontaktiert, setzen Sie sich mit dem Gesundheitsamt in Verbindung. Das Gesundheitsamt kann Ihnen gegebenenfalls eine TAN ausstellen.</li><li>Klicken Sie auf der Startseite der CWA auf „QR-Code erstellen“.</li><li>Klicken Sie auf der „Meine QR-Codes“-Seite oben rechts auf das Drei-Punkte-Menü und dann auf den Menüpunkt „In Vertretung warnen“.</li><li>Klicken Sie auf dem Infoscreen auf „Weiter“.</li><li>Wählen Sie eine Veranstaltung, für welches Sie in Vertretung einer infizierten Person andere warnen möchten. Oder stellen Sie eine gelöschte Veranstaltung wieder her, indem Sie den QR-Code der Veranstaltung erneut einscannen. Klicken Sie auf „Weiter“.</li><li>Geben Sie die Aufenthaltsdauer der positiv getesteten Person an, für die gewarnt werden soll. Diese Information erhalten Sie vom Gesundheitsamt. Klicken Sie auf „Weiter“.</li><li>Geben Sie die 10-stellige TAN ein, die Ihnen mitgeteilt wurde. Klicken Sie auf „Warnung abschicken“.</li></ul>",
```
[the remark this answers is not in the document]
Comment (larswmh) [reply src/data/faq_de.json:1447]: I went with "Klicken Sie unter Status (vor Version 2.12 'Startseite') in Ihrer CWA auf „QR-Code erstellen“." now, thanks for pointing it out

File: src/data/faq.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq.json:1438]: Changed with the next commit, thanks

File: src/data/faq.json [hunk 3]
```diff
@@ -1436,8 +1436,7 @@
...
-                            "<div class='container'><div class='row'><div class='col-sm'><img class='img-fluid' src='../../assets/img/faq/warn_for_others4.png' alt='image'></div><div class='col-sm'><img class='img-fluid' src='../../assets/img/faq/warn_for_others5.png' alt='image'></div><div class='col-sm'><img class='img-fluid' src='../../assets/img/faq/warn_for_others6.png' alt='image'></div></div></div>"
+                            "For exemplification, you can find screenshots for warning for others for both iOS and Android devices at the following links:<ul><li><a href='../screenshots/#ios_warn_for_others' target='_blank'>iOS Screenshots: Warn For Others</a></li><li><a href='../screenshots/#android_warn_for_others' target='_blank'>Android Screenshots: Warn For Others</a></li></ul>"
```
[the remark this answers is not in the document]
Comment (larswmh) [reply src/data/faq.json:1439]: Changed with the next commit, thanks

**Review by MikeMcC399 — APPROVED** [review]
LGTM!

**Review by dsarkar — APPROVED** [review]

**Pull request merged**